### ISSUE TITLE
'Open in Mobile App' completely broken

### ISSUE THREAD

**Issue by lountz** [Ombi-app/Ombi#4286]
**Describe the bug**
On iPhone 12 I am trying to access ombi via my mobile app from safari by clicking 'Open Mobile App' but it launches the app and all the pages are blank and I cannot logout in the mobile app. I tried uninstalling and reinstalling the mobile app to no avail.

**To Reproduce**
1. Open Ombi on mobile browser
2. Click on 'Open Mobile App'
3. See error

**Expected behavior**
The mobile app open and display proper info 
**Screenshots**

**Logs (Logs directory where Ombi is located)**
[log-20210807.txt](https://github.com/Ombi-app/Ombi/files/6949780/log-20210807.txt)

**Desktop (please complete the following information):**
Windows 10

**Ombi Version (please complete the following information):**
 - Version 4.0.1458   
- Plex Media Server
- Database Type: SQLite (Please change if using MySQL)

**Additional context**

**Comment by tidusjar** [comment]
Is your application URL correctly set?

**Comment by lountz** [comment]
I wanted to ask, how specifically am I supposed to set that up? Right now I'm just accessing ombi via my computer's ip with the port number, but when I enter that as the Application url it does not work.

**Comment by tidusjar** [comment]
So the application url should be the externally accessible url for Ombi, this is what the App would communicate with, see here: https://docs.ombi.app/settings/customization/#application-url

**Comment by lountz** [comment]
I saw that, but to add that, do I have to buy a domain and create that myself? I tried putting in a domain similar to the example but it didnt work.

**Comment by tidusjar** [comment]
It can be an external IP address for example

**Comment by lountz** [comment]
I have my pc's external ip address entered for the application url as http://<ipaddress>:<port number>. Is that a correct format? Because it's not working when I do that. It also doesn't work if I remove the port number

**Comment by tidusjar** [comment]
So once you have filled in your application url, you can then re-scan the bar code under your user profile.

An application URL of `http://192.168.1.1:5000` would work fine, but in my example because I am referencing an internal IP, then the app would only be able to work and communicate with Ombi when my mobile device is on the internal network. If you are using an external IP address, ensure you can access it on your mobile by navigating to the external IP in your mobile browser just to ensure it's all good

**Comment by lountz** [comment]
See, that's the issue. I'm able to navigate to the ip just fine from my mobile browser, but when I click 'Open in Mobile App' or scan the qr code, it takes me to a blank screen in the app every time, even with the application url set up to match the ip in my browser.

**Comment by tidusjar** [comment]
For your application URL, ensure that it does not contain `/login` for example it should just look like `http://192.168.1.1:5000` with nothing after it.

**Comment by tidusjar** [comment]
Have you tried logging in manually? That's very strange it's not working

**Comment by lountz** [comment]
Yeah, when I log in manually I get a popup error saying 'Wrong Server Version'  followed by a 'Login Error' popup.

**Comment by tidusjar** [comment]
What happens when you navigate to your IP/api/v1/status/info

Because that's what the app is checking

**Comment by lountz** [comment]
I found that I was using the wrong IP address (local instead of public). That said, the 'open in mobile app' still does not work, but if I enter the same ip in the app manually and login it works, so I think there is still a bug.

**Comment by BourbonCoffee** [comment]
Correct me if I am wrong, but the application URL is only for externally accessible addresses. This would be something like https://ombi.yourdomain.com or [external IP]:[OmbiPort]. The reason the page is blank is because the app loaded but it has no data to load and you have already logged in once on the Ombi app.

In order to fix this you need to either:

1. Buy or use a domain name you already own. Then set up that domain name to point to your external IP or to a DDNS service (like duckdns.org) and use something like ngnix and Swag (LetsEncrypt) to reverse proxy the connection (so you aren't just opening your server to anyone). Then on your router your need to port forward the appropriate ports.
2. Do the same as above, but just with your external IP rather than a domain name.

**If you are not sure what you are doing on the networking end of it, please look up some guides. You _DO NOT_ want to expose your server to the internet!**
